oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-tools-file https://github.com/akretion/server-tools.git 8-attachm-meta
```

Comment:
The branch 8-attachm-meta was proposed in this pull request :  
https://github.com/OCA/server-tools/pull/376